File: VB/Console/MultipleInheritance.vb
interface A
	
	sub show()
End interface

interface B
	
	sub show()
end interface


class C
	Implements A, B
	Dim c as integer
	
	public sub new()
		c=30
	end sub
	public sub show() implements A.show, B.show
		Console.WriteLine("c=" & c)								'Overriding
	end sub
end class


Module MultipleInheritanceTest
	Sub Main()
		Dim obc as new C()
	
		obc.show()
	End Sub
End Module

File: CS/Console/MultipleInheritance.cs
using System;

interface A
{
	
	public const int a=10;
	
	void show();
}

interface B
{
	const int b=20;
	
	void show();
}


class C : A, B
{
	int c;
	
	public C()
	{
		c=30;
	}
	public void show()
	{
		Console.WriteLine("c="+c);		//Overriding
	}
}


class MultipleInheritanceTest
{
	static void Main()
	{
		C obc = new C();
	
		obc.show();
	}
}
